vivisect/tools/graphutil.py

```diff
@@ -791,6 +763,8 @@ def getFuncCbRoutedPaths_genback(self, fromva, tova, loopcnt=0, maxpath=None, ma
                 npath = vg_pathcore.newPathNode(parent=cpath, nid=fromid, eid=None)
                 todo.append((fromid,npath))
 
+        self.__go__ = False
+
     def getFuncCbRoutedPaths(self, fromva, tova, loopcnt=0, maxpath=None, maxsec=None):
```

Comment:
it's not part of this delta but...   if maxsec is a relative offset timeout ( which is how it works here ) it should really be "timeout" ( for consistency ).  I usually reserve max/min prefixes for variables that are actual static min/max values, not relative offsets :)